d2renderers/d2svg/table.go

```diff
@@ -91,6 +121,8 @@ func drawTable(writer io.Writer, targetShape d2target.Shape) {
 	rectEl.Fill, rectEl.Stroke = d2themes.ShapeTheme(targetShape)
 	rectEl.ClassName = "shape"
 	rectEl.Style = targetShape.CSSStyle()
+	rectEl.Rx = float64(targetShape.BorderRadius)
```

Comment:
@alixander after add `Rx` and `Ry` attributes for sql_table and class rect, lots of testcases fails, i think regenerate these testcase should be the correct way right? or do i need to add these attributes when `Shape.BorderRadius` not equal to zero?